launch_testing_ros/examples/check_msgs_launch_test.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2021 Open Source Robotics Foundation, Inc.
```

Comment:
It would be great to actually run these tests when testing the package.
Maybe a symlink in the test folder solves the issue (?).

If we move the other example [here](https://github.com/ros2/launch_ros/tree/master/launch_testing_ros/test/examples), we could symlink the complete examples folder.

Reply:
I'm worried if symlinks would work correctly on windows CI. Maybe it is safer to move the files themselves

Reply:
Moved the examples, CI can't find ``demo_nodes_cpp``